### ISSUE TITLE
/eth/v1/validator/synccommitteecontribution returns non-stringified integer

### ISSUE THREAD

**Issue by mcdee** [sigp/lighthouse#2514]
## Description

The API endpoint `/eth/v1/validator/synccommitteecontribution` returns an incorrect type for `subcommittee_index`

## Version

1.5.0-rc0

## Present Behaviour

Data returned is:

```
{"data":{"slot":"20410","beacon_block_root":"0x01f7c37b41e9e632a1c234310de961219712891a37e0544ede77df994069afd3","subcommittee_index":1,"aggregation_bits":"0x00040000c00000000200010002000000","signature":"0xb45858bb0fb05b209e80382961374215314f8b86d880bb69a0adcdb1e6a77b34cd9ce4e8ae8494166862823bf9ad04f913d2f380c498af4c0bdb08ef3bc042860bd5bd37d009fcff29cd024d97b56327dcc311e4e0c5698674f110711aed03b5"}}
```

## Expected Behaviour

The `subcommittee_index` value is returned as a number _i.e._ `1`.  It should be returned as string-quoted, _i.e._ `"1"`.

**Comment by michaelsproul** [comment]
Great catch! Thank you.

We can likely fix this for v1.5.0

**Comment by michaelsproul** [comment]
Fixed in https://github.com/sigp/lighthouse/pull/2515 which will ship in v1.5.0-rc.1
